doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -236,9 +236,10 @@ Please note that "shared" is a reference to multi-tenant support and pooled stor
 | `req.kcm.02` | General | The Architecture **must** support discoverability of nodes and their features. |
 | `req.kcm.03` | General | The Architecture **must** support scheduling of workloads based on Enhanced Platform Awareness (EPA) features such as CPU Pinning, huge-pages and SR-IOV. |
 | `req.kcm.04` | General | The Architecture **must** include kubernetes artefacts (e.g., images, Helm charts, etc.) repository capabilities. |
-| `req.kcm.05` | General | The Architecture **should** support multi-tenancy in Kubernetes cluster. |
+| `req.kcm.05` | General | The Architecture **should** prevent workloads from interfering with or observing other mutually ignorant workloads. |
 | `req.kcm.06` | General | The Architecture **must** support resource tagging. |
 | `req.kcm.08` | General | The Architecture **must** support workload resiliency. |
+| `req.kcm.09` | General | The Architecture **should** prevent workloads from starving other workloads of guaranteed resources. |
```

Comment:
__Query__ - Why not a must?

Reply:
This comment is not relevant for the change under review. @deebhatia  can you please create an issue about this [here](https://github.com/cntt-n/CNTT/issues)?

Reply:
Raised. https://github.com/cntt-n/CNTT/issues/1132

Reply:
@deebhatia agh sorry, I did not realize that line 242 was also a new line added. My bad. Your comment is relevant here. 
I also agree that it should be a must (however Kubernetes supports this without any additions)

Reply:
no worries @CsatariGergely.